### ISSUE TITLE
TextInput: Support onKeyDown callback

### ISSUE THREAD

**Issue by appden** [necolas/react-native-web#582]
**Do you want to request a *feature* or report a *bug*?**

Feature

**What is the current behavior?**

Passing `onKeyDown` into `TextInput` is silently ignored.

**What is the expected behavior?**

Although not supported by React Native, it would be great to support `onKeyDown` for the web. My particular use case is I need to have special behavior for the `Tab` and `Shift+Tab` keys in certain circumstances. That means I need the ability to `preventDefault()` in those situations as well. I'm able to workaround this limitation of `TextInput`, so it isn't very high priority for me, but it's definitely a nice-to-have. Thanks!

**Comment by necolas** [comment]
I've made a change so that `Tab` calls `onKeyPress` and `preventDefault` is available on the event. Hopefully that helps you with your use case.

**Comment by appden** [comment]
@necolas I forgot to thank you for addressing this issue so quickly. So thank you!!!!

Interestingly I found still need to use my `onKeyDownCapture` workaround for capturing the Tab key because React attaches this listener to the `document` and I actually need to `stopPropagation()` in the capture phase because once React calls `onKeyDown` in the bubble phase it's too late (the internal listener handles the Tab key). I'm not asking for you to spend any more time on this, but I did want to let you know what I found out. Thanks again!
